packages/@react-aria/focus/stories/FocusScope.stories.tsx

```diff
@@ -61,6 +61,7 @@ function NestedDialog({onClose, usePortal, contain}: {onClose: VoidFunction, use
     <MaybePortal usePortal={usePortal}>
       <FocusScope contain={contain} restoreFocus autoFocus>
         {!showNew && (
+          // eslint-disable-next-line jsx-a11y/no-noninteractive-element-interactions
```

Comment:
i assume this is just lint cleanup?

Reply:
Yes. The test is listening for `onKeyDown` on a non-interactive element.